frank/utilities.py

```diff
@@ -64,78 +64,169 @@ class UVDataBinner(object):
 
     def __init__(self, uv, V, weights, bin_width):
         nbins = np.ceil(uv.max() / bin_width).astype('int')
+        #Protect against rounding
+        if nbins * bin_width < uv.max():
+            bins += 1
```

Comment:
this line should cause undefined reference: `bins` is not defined  before. 
I guess it's a typo and should be
```suggestion
            nbins += 1
```

Reply:
I guess the tests are not testing this case otherwise they would have failed.
Perhaps worth adding a test, just a simple copy-paste of a test you already have to see that nbins is actually incremented.